conda.recipe/pybert/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "pybert" %}
-{% set version = "4.0.0" %}
+{% set version = "4.0.4" %}
```

Comment:
to keep pipBERT happy, gotta update the version in the `__init__.py` file too.  I'm pushing a commit as such.

Reply:
The name change from `PyBERT` to `PipBERT` also has a fun side effect that click can't deduce out the version number which will erroneously make that unit test fail for far too long than I care to admit.

Reply:
> to keep pipBERT happy, gotta update the version in the `__init__.py` file too. I'm pushing a commit as such. We might look at using [bump2version](https://github.com/c4urself/bump2version) or something with version numbers in 3-4 different places.

Yes! This has bugged me for a _long_ time.
Have you used the package you're proposing, previously?

Reply:
> The name change from `PyBERT` to `PipBERT` also has a fun side effect that click can't deduce out the version number which will erroneously make that unit test fail for far too long than I care to admit.

D'oh! Shoot, the guy, who stole the name "PyBERT" on PyPi, hasn't responded to my request to have it back. I fear we're stuck w/ "PipBERT". Is there a work-around for this?